src/commands/info.js

```diff
@@ -1,16 +1,45 @@
 // @flow
 import * as options from '../utils/options';
 import { BoltError } from '../utils/errors';
+import * as yarn from '../utils/yarn';
 
-export type InfoOptions = {};
+export type InfoOptions = {|
+  cwd?: string,
+  args: options.Args
+|};
+
+function flagsToArgs(flagArgs: options.Flags): Array<string> {
+  const flagArgsKey = Object.keys(flagArgs);
+  // Flags has default -- key
+  if (flagArgsKey.length < 1) {
+    return [];
+  }
+  const flag = [];
+  flagArgsKey.forEach(key => {
+    if (flagArgs[key] === true) {
+      flag.push(`--${key}`);
```

Comment:
flags received here are converted to camelCase by meow, therefore, they may not pass down to yarn same as what bolt receives, if json is the only flag supported (AFAIK) we can capture it explicitly.